src/views/SingleEventView.vue

```diff
@@ -329,7 +354,28 @@ export default {
   font-weight: bold;
 }
 
+.announcements-list-container {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  margin-top: 12px;
```

Comment:
⛏️ use shorthand `margin: 12px 12px 12px 36px` (top right bottom left)

(there might be a better way than above too)